### PR TITLE
Measure Metadata length by grapheme

### LINKED ISSUE #2022: wikipedia_hi_medicine has improper title
Recipe: https://farm.openzim.org/recipes/wikipedia_hi_medicine

```
**********

MetaData Title: must NOT have more than 30 characters

**********
```

I disabled the recipe for now.

### PR DESCRIPTION
**Pull request by kelson42**
Fixes #2022

### CODE REVIEW

File: src/util/metaData.ts [hunk 1]
```diff
@@ -18,18 +21,29 @@ ajv.addKeyword({
...
     Name: { type: 'string', minLength: 1 },
     Creator: { type: 'string', minLength: 1 },
-    Description: { type: 'string', maxLength: 80, minLength: 1 },
+    Description: { type: 'string', uMaxLength: 80, minLength: 1 },
     Language: { type: 'string', minLength: 1, pattern: '^\\w{3}(,\\w{3})*$' },
     Publisher: { type: 'string', minLength: 1 },
-    Title: { type: 'string', maxLength: 30, minLength: 1 },
-    Date: { type: 'string', maxLength: 10, minLength: 10 },
+    Title: { type: 'string', uMaxLength: 30, minLength: 1 },
+    Date: { type: 'string', uMaxLength: 10, minLength: 10 },
```
Comment (veloman-yunkan) [inline src/util/metaData.ts:44]: I think that for the `Date` field (which should be ASCII) there is no need to measure length by grapheme count

File: src/util/metaData.ts [hunk 2]
```diff
@@ -1,4 +1,7 @@
 import AjvModule from 'ajv'
+import type { KeywordCxt } from 'ajv'
+import { byGrapheme } from 'split-by-grapheme'
+import * as logger from '../Logger.js'
```
Comment (veloman-yunkan) [inline src/util/metaData.ts:4]: 'logger' is defined but never used

This is a finding by GitHubActions. There are a few other issues reported by GitHubActions and I believe that some of them should be fixed.

File: src/util/metaData.ts (hunk 1 above)
Comment (kelson42) [reply src/util/metaData.ts]: yes, fixed

File: src/util/metaData.ts (hunk 2 above)
Comment (kelson42) [reply src/util/metaData.ts]: yes, fixed.

**Comment by kelson42** [comment]
@veloman-yunkan Thx for the review, waiting feedback from the requester because I have changed the values of the accesskeys.

**Comment by veloman-yunkan** [comment]
> @veloman-yunkan Thx for the review, waiting feedback from the requester because I have changed the values of the accesskeys.

@kelson42 I think that you confuse this PR with kiwix/libkiwix#1075

**Pull request merged**